docker/README.md

```diff
@@ -15,15 +15,26 @@ software development. Visit
 <a href="https://www.docker.com/whatisdocker" target="_blank">https://www.docker.com/whatisdocker</a> to read more about
 how docker can drastically simplify development and deployment.
 
-## There are two available Docker images for Newman
-### Alpine 3.3 (lightweight):
+## There are four available Docker images for Newman
+### Alpine (lightweight):
+   * <a href="https://hub.docker.com/r/postman/newman_alpine/">DockerHub</a>
+   * <a href="https://github.com/postmanlabs/newman/tree/develop/docker/images/alpine">Documentation</a>
+
+### Ubuntu:
+   * <a href="https://hub.docker.com/r/postman/newman_ubuntu/">DockerHub</a>
+   * <a href="https://github.com/postmanlabs/newman/tree/develop/docker/images/ubuntu">Documentation</a>
+
+### Alpine 3.3 (lightweight, deprecated):
    * <a href="https://hub.docker.com/r/postman/newman_alpine33/">DockerHub</a>
    * <a href="https://github.com/postmanlabs/newman/tree/develop/docker/images/alpine33">Documentation</a>
 
-### Ubuntu 14.04:
+### Ubuntu 14.04 (deprecated):
    * <a href="https://hub.docker.com/r/postman/newman_ubuntu1404/">DockerHub</a>
    * <a href="https://github.com/postmanlabs/newman/tree/develop/docker/images/ubuntu1404">Documentation</a>
 
+### Note:
+The Docker images Alpine 3.3 and Ubuntu 14.04 for Newman are deprecated in the upcoming Newman versions. It is advised to use the Docker images postman/newman_alpine and postman/newman_ubuntu for Newman.
```

Comment:
`in the upcoming Newman versions` is not needed. Specify exact names in place of `Alpine 3.3` and `Ubuntu 14.04`, as well as in the headings above.